Patches/Rim-Sheks!/Patch_PawnKinds_Sheks.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>	
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Rim-Shek</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/PawnKindDef[defName="Shek_Knight"]/combatPower</xpath>
+					<value>
+						<combatPower>60</combatPower>
+					</value>
+				</li>
+
+				<!-- SHEK OUTLANDERS???Why do they override the vanilla defs?-->
```

Comment:
This issue appears to have been fixed, and they're now separate. They'll need loadouts assigned.

Reply:
> This issue appears to have been fixed, and they're now separate. They'll need loadouts assigned.

I checked the new xml files and nothing has changed. Pirate and Town guard is being overridden by Shek race for some unknown reason.